src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -340,7 +352,7 @@ private void handleBegin(
                     tlsParameters.setServerNames(asList(new SNIHostName(tlsHostname)));
                 }
 
-                String[] applicationProtocols = config.clientApplicationProtocols();
+                String[] applicationProtocols = new String[] { applicationProtocol };
                 if (applicationProtocols.length > 0)
                 {
```

Comment:
The code logic has changed here. What is the expected behavior when `tlsApplicationProtocol` is `null`?

Also need to remove the unnecessary `try-catch` around `tlsParameters.setApplicationProtocols(...)` given that the TLS nukleus requires Java 9 in order to run anyway.

Reply:
try-catch is not there anymore.